server/test/api/costCenter.js

```diff
@@ -33,28 +33,61 @@ describe('The cost center API, PATH : /cost_centers', function () {
        .catch(helpers.handler);
     });
 
-  it('METHOD : GET, PATH : /cost_centers, It returns a list of cost centers', function () {
-    return agent.get('/cost_centers')
+    it('METHOD : GET, PATH : /cost_centers?availableOnly=1, It returns a list of availables cost centers', function () {
+      return agent.get('/cost_centers?availableOnly=1')
```

Comment:
Why not just call this query string parameter "available"?  Also, technically, query strings are case-sensitive, but this isn't always a guarantee.  We should prefer not using case in our query parameters.  ExpressJS is by default case insensitive for its routes [src](http://expressjs.com/en/api.html).